detectable/src/main/java/com/synopsys/integration/detectable/detectables/lerna/LernaMissingDependencyHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.synopsys.integration.detectable.detectables.lerna;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+
+import com.synopsys.integration.bdio.graph.builder.LazyExternalIdDependencyGraphBuilder;
+import com.synopsys.integration.bdio.model.Forge;
+import com.synopsys.integration.bdio.model.dependencyid.DependencyId;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import com.synopsys.integration.bdio.model.externalid.ExternalIdFactory;
+import com.synopsys.integration.detectable.detectables.lerna.model.LernaPackage;
+import com.synopsys.integration.detectable.detectables.npm.lockfile.model.NpmRequires;
+import com.synopsys.integration.detectable.detectables.npm.lockfile.parse.NpmLockfilePackager;
+import com.synopsys.integration.detectable.detectables.yarn.parse.YarnTransformer;
+
+public class LernaMissingDependencyHandler {
+    private final List<LernaPackage> lernaPackages;
+
+    public LernaMissingDependencyHandler(List<LernaPackage> lernaPackages) {
+        this.lernaPackages = lernaPackages;
+    }
+
+    public void missingNpmDependencyHandler(Logger logger, NpmRequires missingDependency) {
+        boolean isLernaPackage = lernaPackages.stream()
+                                     .anyMatch(lernaPackage -> lernaPackage.getName().equalsIgnoreCase(missingDependency.getName()));
+        if (!isLernaPackage) {
+            NpmLockfilePackager.handleMissingDependency(logger, missingDependency);
+        }
+    }
+
+    public ExternalId missingYarnDependencyHandler(Logger logger, ExternalIdFactory externalIdFactory, DependencyId dependencyId, LazyExternalIdDependencyGraphBuilder.LazyDependencyInfo lazyDependencyInfo, String yarnLockFilePath) {
```

Comment:
This is definitely not worth delaying the code freeze for, but the noun-like method names caught my eye. Then I saw that they are kinda used like objects (method references), but even so... since they're doing real work, I think I'd lean toward something more verb-like (say, handleMissingXxxDependency())

Reply:
Good catch. That's a quick fix